packages/hoppscotch-common/src/pages/index.vue

```diff
@@ -64,20 +64,40 @@
       @submit="renameReqName"
       @hide-modal="showRenamingReqNameModal = false"
     />
-    <HoppSmartConfirmModal
-      :show="confirmingCloseForTabID !== null"
-      :confirm="t('modal.close_unsaved_tab')"
-      :title="t('confirm.save_unsaved_tab')"
-      @hide-modal="onCloseConfirmSaveTab"
-      @resolve="onResolveConfirmSaveTab"
-    />
     <HoppSmartConfirmModal
       :show="confirmingCloseAllTabs"
       :confirm="t('modal.close_unsaved_tab')"
       :title="t('confirm.close_unsaved_tabs', { count: unsavedTabsCount })"
       @hide-modal="confirmingCloseAllTabs = false"
       @resolve="onResolveConfirmCloseAllTabs"
     />
+    <HoppSmartModal
+      v-if="confirmingCloseForTabID !== null"
+      :title="t('modal.close_unsaved_tab')"
+      @close="confirmingCloseForTabID = null"
```

Comment:
```suggestion
    <HoppSmartModal
      v-if="confirmingCloseForTabID !== null"
      dialog
      role="dialog"
      aria-modal="true"
      :title="t('modal.close_unsaved_tab')"
      @close="confirmingCloseForTabID = null"
```

can you add the aria labels as well, similar to the `HoppSmartConfirmModal.vue`